metadatastore/examples/sample_data/temperature_ramp.py

```diff
@@ -23,10 +27,14 @@ def run(run_start_uid=None, sleep=0):
     # Create Event Descriptors
     data_keys1 = {'point_det': dict(source='PV:ES:PointDet', dtype='number')}
     data_keys2 = {'Tsam': dict(source='PV:ES:Tsam', dtype='number')}
-    ev_desc1_uid = insert_event_descriptor(run_start=run_start_uid,
-                                           data_keys=data_keys1, time=common.get_time())
-    ev_desc2_uid = insert_event_descriptor(run_start=run_start_uid,
-                                           data_keys=data_keys2, time=common.get_time())
+    ev_desc1_uid = insert_descriptor(run_start=run_start_uid,
+                                           data_keys=data_keys1,
```

Comment:
too many spaces before data_keys and the next three lines. same for ev_desc_2_uid